index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
I think for now, we can update this title back to "Mobile app SDK for iOS and Android".

And yeah, would love to add even more metadata! Good idea.